assets/pages/new-contributions-landing/components/ContributionState.jsx

```diff
@@ -7,23 +7,30 @@ import { connect } from 'react-redux';
 
 import { usStates, caStates } from 'helpers/internationalisation/country';
 import { type CountryGroupId } from 'helpers/internationalisation/countryGroup';
+import { classNameWithModifiers } from 'helpers/utilities';
 
 import SvgGlobe from 'components/svgs/globe';
+import { type State } from '../contributionsLandingReducer';
 
 // ----- Types ----- //
 type PropTypes = {
   countryGroupId: CountryGroupId,
   selectedState: string,
 };
 
+const mapStateToProps: State => PropTypes = () => ({
+  countryGroupId: 'UnitedStates',
```

Comment:
Is this being double-passed? It's also being passed in as a prop at the component call-site in `contributionsLanding.jsx`.